var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -66,9 +67,24 @@ class Julia(MakefilePackage):
     depends_on("libuv", when="@:1.7")
     depends_on("libuv-julia@1.42.0", when="@1.8.0:1.8.1")
     depends_on("libuv-julia@1.44.2", when="@1.8.2:1.9")
-    depends_on("libuv-julia@1.44.3", when="@1.10.0:")
+    depends_on("libuv-julia@1.44.3", when="@1.10.0:1.10")
+    depends_on("libuv-julia@1.48.0", when="@1.11.0:")
     depends_on("suite-sparse@5.4:5.10", when="@1.6:1.9")
 
+    with when("@1.11.0:1.11"):
+        # libssh2.so.1, libpcre2-8.so.0, libmbedtls.so.14, libmbedcrypto.so.7, libmbedx509.so.1,
+        # libopenlibm.so.4, libblastrampoline.so.5, libgit2.so.1.6, libnghttp2.so.14,
+        # libcurl.so.4
+        depends_on("libblastrampoline@5.11.0:5")
+        depends_on("libgit2@1.7.2:1.7")
+        depends_on("libssh2@1.11.0:1.11")
```

Comment:
```suggestion
        depends_on("libssh2@1.11")
```

wondering if suite-sparse has to be pinned on a patch version number, or 7.7 is also fine

Reply:
> wondering if suite-sparse has to be pinned on a patch version number, or 7.7 is also fine

I think suite-sparse hasn't traditionally followed semver very closely.  Also, using a version of cholmod different from that [Julia was tested against](https://github.com/giordano/SparseArrays.jl/blob/feb54ee5e49008bd157227099cafe604a67c36fb/src/solvers/wrappers.jl#L3222-L3228) would trigger an [annoying warning](https://github.com/giordano/SparseArrays.jl/blob/feb54ee5e49008bd157227099cafe604a67c36fb/src/solvers/cholmod.jl#L205-L220) at runtime.